install/ci-vm/ci-linux/ci/runCI

```diff
@@ -59,7 +59,6 @@ postStatus "preparation" "Loaded variables and created log file"
 # Truncate log in case there was anything left from a previous session
 echo "" > "${logFile}"
 
-postStatus "building" "Checking if CCExtractor build artifact is present"
```

Comment:
Please change L57 to `postStatus "preparation" "Loaded variables, created log file and checking for latest CCExtractor artefact"`